app/src/main/java/com/asfoundation/wallet/ui/backup/BackupCreationPresenter.kt

```diff
@@ -126,6 +144,9 @@ class BackupCreationPresenter(
   private fun handleSaveAgainClick() {
     disposables.add(view.getSaveAgainClick()
         .doOnNext { activityView.askForWritePermissions() }
+        .doOnNext {
+          walletEventSender.sendWalletConfirmationBackupEvent(WalletAnalytics.ACTION_SAVE)
```

Comment:
This just asks for the permissions the user hasn't saved the wallet here yet
Also, why not on the previous doOnNext